testsuite/forge/src/backend/local.rs

```diff
@@ -123,6 +124,59 @@ impl Swarm for LocalSwarm {
         todo!()
     }
 
+    fn cluster(&mut self) -> Result<Cluster> {
+        let http_client = reqwest::Client::new();
+        let root_key_path = self.validator_swarm.config.diem_root_key_path.clone();
```

Comment:
I don't think we should allow this. One thing I tried very hard to do is make it impossible for tests to gain access to the root key without going through something like the LocalAccount (we might even want to make something a bit more strict than LocalAccount). That way we're careful to only allow access to the Root accounts to one thing at a time. By doing this we risk allowing multiple things to submit txns simultaneously from the root account leading to flakyness